### DISCUSSION TITLE
Need Navbar center align in Bootstrap 5.1.3

### DISCUSSION THREAD

**Discussion by Satbir4** [twbs/bootstrap#35792] [Q&A] [answered]
### Need Navbar center align in Bootstrap 5.1.3
Can anyone send me the code,
i need navbar like the image following

[https://xp.io/storage/19zvM7y0.png](https://xp.io/storage/19zvM7y0.png)

**Comment by gerardnico** [comment, marked as answer]
https://getbootstrap.com/docs/5.0/utilities/spacing/#horizontal-centering
